### PR TITLE
Add detail on charter changes

### PR DESCRIPTION
**Pull request by wseltzer**
Add detail on the Team and Director process for circulating proposed charter changes in response to AC review.

### CODE REVIEW

File: process/charter.html [hunk 1]
```diff
@@ -485,7 +485,9 @@ <h3>5.2 <a id="review-eval" name="review-eval">Evaluating the
...
 
-<p>Very minor changes may be made in place to a charter. If there are only very minor changes to a charter resulting from the review, the (future) Director's decision includes the original charter URI.</p>
+<p>Very minor changes may be made in place to a charter. If there are only very minor changes to a charter resulting from the review, the (future) Director's decision includes the original charter URI and an explanation of the changes and their rationale.</p>
+<p>If substantive changes are proposed in response to charter review, Team must send the charter with proposed changes, a diff, and an explanation of the changes and their rationale to all who reviewed the charter, copying the member-charters-review@w3.org member-archived mailing list, accompanied by a request for response (in no less than one week) if reviewers have concerns or if the changes would alter their reviews. If the work continues or derives from an existing Group or community effort, Team should also send the diff and a public rationale to that Group or community. These communications should note that the Director has not yet approved the charter.</p>
```
Comment (nigelmegitt) [inline process/charter.html]: > request for a response (in no less than one week)

appears to suggest that responses are not permitted within the first week, whereas I think this probably is supposed to say "within a minimum response period no less than one week" or something like that.

File: process/charter.html [hunk 2]
```diff
@@ -485,7 +485,9 @@ <h3>5.2 <a id="review-eval" name="review-eval">Evaluating the
...
+<p>If proposed changes are substantial, or if anyone expresses concern in response to the re-review, Team should formally open a new AC charter review.</p>
```
Comment (plehegar) [inline process/charter.html]: This seems to always require a new AC review if there are substantive changes, which contradicts the paragraph above. If none of the reviewers objects to the change after a week review, why do a new AC review?

Comment (wseltzer) [reply process/charter.html]: I was differentiating between "substantial" (major) and substantive (could be major or minor). For major changes, yes, I was saying they should by default go to new AC review. 

Should I remove that clause?

Comment (wseltzer) [reply process/charter.html]:
```suggestion
<p>If anyone expresses new concern in response to the re-review, Team may attempt to resolve the concern (with re-review), formally open a new AC charter review, or the Director may treat it as an objection and overrule it.</p>
```

**Pull request merged**